modules/samples/consumers/kafka-performance/build.xml

```diff
@@ -0,0 +1,80 @@
+<!--
+  ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project default="run">
+
+    <property name="carbon.home" value="../../.."/>
+    <property name="src.dir" value="src/main/java"/>
+    <property name="resources.dir" value="src/main/resources"/>
+    <property name="temp.dir" value="temp"/>
+    <property name="lib.dir" value="../../lib"/>
+    <property name="class.dir" value="${temp.dir}/classes"/>
+    <property name="main-class" value="org.wso2.carbon.sample.kafka.performance.KafkaConsumer"/>
+
+    <property name="username" value="admin"/>
+    <property name="password" value="admin"/>
+
+    <property name="zookeeperUrl" value="127.0.0.1:2181"/>
+    <property name="groupId" value="group"/>
+    <property name="topic" value=""/>
+    <property name="noOfConsumers" value=""/>
+
```

Comment:
@thiliA 

Better to keep a default value for these parameters in the build xml, so that the script will run with just the 'ant' command. Then in the startup, print all the parameters possible so user knows how to configure them. WDYT?